### ISSUE TITLE
Formatting doesn't preserve comments or empty lines

### ISSUE THREAD

**Issue by byrmsh** [noperator/sol#2]
The current formatting removes comments and empty lines. This is most likely something you're already aware of, @noperator, but I’m creating this issue on the slim chance it hasn’t come up yet.

**Comment by noperator** [comment]
Thanks—I'm aware of comment (and I suppose newline) removal, but haven't tried to solve for it yet. Can you tell me more about your use case? e.g., are you running sol on an entire file (vs. individual one-liners)?

**Comment by byrmsh** [comment]
Yes, in my use case, I am using it in Neovim with conform.nvim with format-on-save for entire bash scripts I edit but as of now, I don't think I can use it in this setup as it removes comments and empty lines. I think I can still use it for contained lines though, such as auto-generated one-line long cURL commands.
